### DISCUSSION TITLE
Is there a 10.8.1 Template available for Enterprise Base deployment?

### DISCUSSION THREAD

**Discussion by BGamified** [Esri/arcgis-cookbook#342] [Q&A] [answered]
Hi there,

- I downloaded version 4.0.0 and noticed there is no 10.8.1 available under templates/arcgis-enterprise-base, only 10.9, 10.9.1 and 11.0 are available.
- I made a copy of the 10.9.1 folder for windows and edited all seven JSON files' references to "10.9.1" to become "10.8.1".
- Can anyone help confirm this is going to work?
- The reason I'm not deploying Enterprise 10.9 and above is due to the need to use ArcFM/ArcMap 10.8.1 on the same machine, and it does not look like ArcGIS Desktop 10.8.1 is supported by ArcGIS Server 10.9.x
- It says: "_ArcGIS Server 10.8.1 and 10.8 can be installed on the same machine as ArcGIS Desktop 10.8.1 and 10.8. This includes ArcReader and ArcGIS Engine clients_._" on https://desktop.arcgis.com/en/arcmap/latest/get-started/introduction/arcmap-108x-and-enteprise.htm

Thanks!

**Comment by cameronkroeker** [comment, marked as answer]
Hi @BGamified,

This is a great question. Yes, in theory the templates should work fine with 10.8.1. The reason the templates only go back to version 10.9 is because that was the first version the templates were released/certified to work with. However, I do not see anything in the templates that would prevent them from working with 10.8.1.

If you run into any issues feel free to reach out and we will do our best to help.

Thanks,
Cameron K.

**Reply by BGamified** [reply]
Thanks Cameron, will post an update here once I've tested it. :)
